### PR TITLE
Add cat details

### PR DESCRIPTION
**Pull request by MissMelisa**
Add a new route for extra data in every cat

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,6 +22,7 @@ function App() {
             <Route path="/catPresentation" exact>
               <CatPresentationPage />
             </Route>
+            <Route path="/:id" children={<CatDetailsPage />}></Route>
```
Comment (falecci) [inline src/App.js:25]:
```suggestion
            <Route path="/:id">
                <CatDetailsPage />
            </Route>
```

You can always pass `children` prop inside the component, like here:

```tsx
            <Route path="/catPresentation" exact>
              <CatPresentationPage />
            </Route>
```

File: src/Components/CatDetails/index.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+import { makeStyles } from "@material-ui/core/styles";
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 0,
+    paddingTop: "56.25%",
```
Comment (falecci) [inline src/Components/CatDetails/index.jsx:16]: Uhm... I don't like this. There must be a better way to achieve what you are trying to do

File: src/Pages/CatDetails/index.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  mainContainer: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+});
+function CatDetailsPage() {
+  const classes = useStyles();
+  let { id } = useParams();
+  const { isLoading, data = [], error } = useQuery("catdata", () =>
```
Comment (falecci) [inline src/Pages/CatDetails/index.jsx:18]: Can we use `catdata` and the breed id as the key for this query?

**Review by falecci — CHANGES_REQUESTED** [review]

**Comment by falecci** [comment]
Looks good! Just a small fixes and ready to go!

File: src/Pages/CatDetails/index.jsx (hunk 3 above)
Comment (MissMelisa) [reply src/Pages/CatDetails/index.jsx:18]: fix it

**Review by falecci — APPROVED** [review]

**Pull request merged**